CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- Repair build scripts to output an es5-compatible module
+
+## Version 1.0.0 - 2018-10-09
```

Comment:
the 1.0.0 release wasn't added to the changelog. I'll clean this up further in a release commit.